client/landing/jetpack-cloud/components/daily-backup-status/index.jsx

```diff
@@ -66,19 +67,22 @@ class DailyBackupStatus extends Component {
 
 		const displayDate = this.getDisplayDate( backup.activityTs );
 
+		const meta = get( backup, 'activityDescription[2].children[0]', '' );
+
 		return (
-			<Fragment>
-				<Gridicon className="daily-backup-status__status-icon" icon="cloud-upload" />
-				<div className="daily-backup-status__label">
-					{ translate( 'Latest backup completed:' ) }
+			<Card className="daily-backup-status__success">
+				<div className="daily-backup-status__icon-section">
+					<Gridicon className="daily-backup-status__status-icon" icon="cloud-upload" />
+					<div className="daily-backup-status__title">{ translate( 'Latest backup' ) }</div>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.